layouts/partials/home.html

```diff
@@ -18,7 +18,8 @@ <h1 class="greeting">Hi, I am {{ if .Site.Params.author.name }}{{ .Site.Params.a
       <li>{{ . }}</li>
       {{ end }}
     </ul>
-    <!-- @todo this section should go to the first section, not necessarily about -->
-    <a href="#about"><i class="arrow bounce fa fa-chevron-down"></i></a>
+    {{ range first 1 (sort .Site.Data.sections "section.weight") }}
```

Comment:
You need to check if the `.Site.Data.sections` exist or not.

```html
{{ if .Site.Data.sections }}
 {{ range first 1 (sort .Site.Data.sections "section.weight") }}
    <a href="#{{ replace (lower .section.name) " " "-" }}"><i class="arrow bounce fa fa-chevron-down"></i></a>
 {{ end }}
{{ else }}
   <a href="#"><i class="arrow bounce fa fa-chevron-down"></i></a>
{{ end }}
```

Reply:
Updated with a check. If the sections don't exist, the chevron is hidden.